reframe/frontend/cli.py

```diff
@@ -45,6 +45,40 @@ def list_checks(checks, printer, detailed=False):
     printer.info('Found %d check(s).' % len(checks))
 
 
+def partition_supported(check, partition, options):
```

Comment:
this function `partition_supported` and the below function `environ_supported` are duplicates of the corresponding functions in the class `Runner`. Once Christopher has finished the issue #488 where he is abstracting the test selection phase of the frontend in a separate class, we may move this two functions to this new class and remove them from here and from the class `Runner`